packages/@aws-cdk/aws-ecs/test/test.container-definition.ts

```diff
@@ -281,15 +281,16 @@ export = {
       ]
     }));
 
-    expect(stack).to(haveResource('AWS::IAM::Policy', {
+    expect(stack).to(haveResourceLike('AWS::IAM::Policy', {
```

Comment:
IAM Policies should use `haveResource` and be complete examples... Otherwise it's easy to allow "more permissive" things through.